.changeset/curvy-colts-applaud.md

```diff
@@ -0,0 +1,5 @@
+---
+"@hashicorp/design-system-components": patch
+---
+
+Upgraded Ember.js to latest stable release `5.12`
```

Comment:
[suggestion] since there are no actual user-facing changes at this point, the changelog could be removed.